middleman-core/lib/middleman-core/step_definitions/server_steps.rb

```diff
@@ -72,60 +72,61 @@
 
 When /^I go to "([^\"]*)"$/ do |url|
   in_current_dir do
-    @browser.get(URI.encode(url))
+    visit(URI.encode(url).to_s)
   end
 end
 
 Then /^going to "([^\"]*)" should not raise an exception$/ do |url|
   in_current_dir do
-    expect{ @browser.get(URI.encode(url)) }.to_not raise_exception
+    
+    expect{ visit(URI.encode(url).to_s) }.to_not raise_exception
   end
 end
 
 Then /^the content type should be "([^\"]*)"$/ do |expected|
   in_current_dir do
-    expect(@browser.last_response.content_type).to start_with(expected)
+    expect(page.response_headers['Content-Type']).to be_start_with expected
```

Comment:
No, this calls `page.response_headers['Content-Type'].start_with?`. But maybe there's a matcher for this, so `be_` can be removed.